covsirphy/cleaning/linelist.py

```diff
@@ -199,13 +199,11 @@ def subset(self, country, province=None):
         df = df.drop([self.COUNTRY, self.PROVINCE], axis=1)
         return df.reset_index(drop=True)
 
-    def closed(self, country, province=None, outcome="Recovered"):
+    def closed(self, outcome="Recovered"):
         """
-        Return subset of the country/province and start/end date.
+        Return subset of global outcome data (recovered/fatal).
```

Comment:
Please also update "Returns" section of the docstring because "country" and "province" will be shown the output.

Reply:
What do you mean? With this return docstring line I changed `the country/province and start/end date` to `global outcome data (recovered/fatal)`. What section do you mean?

Reply:
Please add two lines between 213 and 214.

```
Columns:
    - Country (str): country name
    - Province (str): province name or "-"
    - Hospitalized_date (pandas.TimeStamp or NT)
```

Then, please remove line 227 (+2) `# Subset by area`.